cns/middlewares/k8sSwiftV2_windows.go

```diff
@@ -40,5 +40,12 @@ func (k *K8sSWIFTv2Middleware) assignSubnetPrefixLengthFields(podIPInfo *cns.Pod
 		},
 		GatewayIPAddress: interfaceInfo.GatewayIP,
 	}
+	// assign default route
+	route := cns.Route{
+		IPAddress:        "0.0.0.0/0",
```

Comment:
Why 0.0.0.0/0

Reply:
is this added for linux path though in current scenario?

Reply:
yes, it's added for Linux path, we should do the same thing for windows path